mars-cli/mars_cli.py

```diff
@@ -188,12 +188,6 @@ def cli(ctx, development):
     multiple=True,
     help="Path of files to upload",
 )
```

Comment:
This probably got committed by mistake and can be left in.
```suggestion
)
@click.option(
    "--data-submit-to-ena",
    type=click.BOOL,
    default=False,
    help="Submit data files to ENA.",
)
```